torch/_dynamo/variables/builder.py

```diff
@@ -490,6 +504,26 @@ def tensor_should_specialize(self):
             )
         )
 
+    def wrap_sym(self, value: Union[torch.SymInt, torch.SymFloat]):
+        if not is_constant_source(self.get_source()):
+            self.tx.output.graphargs.append(GraphArg(self.get_source(), value, False))
+        if is_constant_source(self.get_source()):
+            return self.tx.output.register_attr_or_module(
+                value,
+                re.sub(r"[^a-zA-Z0-9]+", "_", self.name),
```

Comment:
It's weird that you have to manually re.sub here

Reply:
Yes, but 
<img width="499" alt="image" src="https://user-images.githubusercontent.com/4755252/201240211-e425ced7-82c0-4c16-a46d-177c0f04d4c3.png">
This is our pattern, for better or worse, and I am choosing consistency for now.